### PR TITLE
LL-6656 - Rebrand Tag Component

### PR DESCRIPTION
**Pull request by jelbaz-ledger**
Add a Tag component.

https://user-images.githubusercontent.com/86958797/129734345-3f294970-528e-4381-a3ad-3cf5a5d13412.mp4

### Type

Feature

### Context

[LL-6656]

[LL-6656]: https://ledgerhq.atlassian.net/browse/LL-6656

### CODE REVIEW

File: ui-lib/src/components/Tag/index.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  type?: "plain" | "translucent" | "outlined";
+}>;
+
+function getColor({ type, active }: Props) {
+  switch (type) {
+    case "translucent":
+    case "outlined":
+      return "palette.v2.primary.Dark";
+    default:
+      return active ? "palette.v2.background.default" : "palette.v2.primary.Dark";
```
Comment (juan-cortes) [inline ui-lib/src/components/Tag/index.tsx:23]: Why is this color uppercase? 🤔

**Review by juan-cortes — DISMISSED** [review]

File: ui-lib/src/components/Tag/index.tsx (hunk 1 above)
Comment (jelbaz-ledger) [reply ui-lib/src/components/Tag/index.tsx:23]: The key is currently uppercased in the palette in the develop branch : https://github.com/LedgerHQ/ledger-live-desktop/blob/develop/ui-lib/src/styles/palettes/light.json#L27

@jgiamporcaro-ledger made a fix but it's still not merged yet I think ❓

Comment (jgiamporcaro-ledger) [reply ui-lib/src/components/Tag/index.tsx:23]: Yeah, currently pending. I'm going to isolate the fix by cherry-pinking it from my branch and send a fix PR. Please review it asap and we'll be able to merge it.

**Review by jgiamporcaro-ledger — DISMISSED** [review]

**Comment by ggilchrist-ledger** [comment]
LGTM 👍

**Pull request merged**